src/Microsoft.DotNet.Cli.Utils/Command.cs

```diff
@@ -131,12 +132,11 @@ public CommandResult Execute()
 
             Reporter.Verbose.WriteLine($"Process ID: {_process.Id}");
 
-            var threadOut = _stdOut.BeginRead(_process.StandardOutput);
-            var threadErr = _stdErr.BeginRead(_process.StandardError);
-
+            var taskOut = _stdOut.BeginRead(_process.StandardOutput);
+            var taskErr = _stdErr.BeginRead(_process.StandardError);
             _process.WaitForExit();
-            threadOut.Join();
-            threadErr.Join();
+
+            Task.WhenAll(taskOut, taskErr);
```

Comment:
Is it `Task.WhenAll(taskOut, taskErr).Wait()`? Because you are not storing the returned `Task` for use anywhere else.